Bottom: e29b651b6873624f9752364354df44f87b19df72
Top:    d1f6bae5852fc416630f1448050162e6a70879b4
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-06 21:11:38 +0000

find.c: The User varaible needs to die...

Reduce its usage to where absolutely necessary... oh, what a surprise,
another variable passing state down into add_object_to_list ()


---

diff --git a/src/find.c b/src/find.c
index 1ef631b..3bd66f6 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3093,7 +3093,7 @@ LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int flag,
    */
   ListStart (type, ptr1, ptr2, ptr3, flag);
   DoIt (flag, AndRats, AndDraw);
-  if (User)
+  if (AndDraw)
     IncrementUndoSerialNumber ();
   User = false;
